src/aspire/basis/fb_2d.py

```diff
@@ -128,26 +130,27 @@ def norms(self):
         """
         Calculate the normalized factors of basis functions
         """
-        norms = np.zeros(np.sum(self.k_max))
+        norms = np.zeros((2, np.sum(self.k_max)))
```

Comment:
Can we just have two variables `radial_norms, angular_norms`? That way we don't have to remember which is `0` and which is `1`.

Reply:
Done in new commit fc4e4c8.